public/pages/Overview/utils/helpers.ts

```diff
@@ -240,6 +247,19 @@ export function getAlertsVisualizationSpec(
     domain: defaultScaleDomain,
   }
 ) {
+  const isGroupedByStatus = groupBy === 'status';
+  let severities = severityOptions.map((severity) => severity.text);
+  severities.reverse().pop();
+
+  let states = ['ACTIVE', 'ACKNOWLEDGED'];
+  const statusColors = {
+    euiColorVis6: '#B9A888',
+    euiColorVis9: '#E7664C',
```

Comment:
@jovancacvetkovic Active should be `euiColorVis9` and acknowledged should be `euiColorVis6`.
Incorrect active color in screenshot:
![image](https://user-images.githubusercontent.com/3692324/211888876-06ded13b-57d2-407c-89d4-4b4f1456bc56.png)